src/App/Game/Game.tsx

```diff
@@ -110,7 +110,13 @@ export default class Game extends React.Component<IGameProps, {}> {
         clientConfig.enhancer = state.config.enhancer;
       }
       if (this.loadAI) {
-        clientConfig.ai = state.ai.bgioAI(aiLevel);
+        if (typeof state.ai === 'undefined') {
+          this.load().then(() => {
```

Comment:
I think this is not addressing the root cause.... The idea is that it would only come up here AFTER the load, given that we have the "loading" flag being false

Reply:
The solution should be:

- Revert these changes you made
- Call this.clear() first thing in the constructor OR load() (I think load() will work while constructor might not)

Reply:
I tried this and tests broke: 
https://semaphoreci.com/freeboardgame/freeboardgame-org/branches/issue-152/builds/2

Not sure whether things actually broke, or whether the tests need to be changed.